Homework5_SvetlanaNesenchuk/Task5/RegularExpressions.py

```diff
@@ -0,0 +1,35 @@
+"""Task5."""
+
+import re
+
+# task 1.1
+string = 'svetlana_nesenchuk@epam.com bedjhfvuyew ' \
+         'kiannet@gmail.com frjhgfru abc@gmail.com'
+print(re.findall(r'\w+@\w+\.\w+', string))
+
+
+# task 1.2
+string = 'qwwertyuuio hjdfh bhr byey beby etr'
+all_words = re.findall(r'\w+', string)
+for word in all_words:
+    if len(word) in (3, 4, 5):
```

Comment:
you can use {3,5} instruction instead in your regular expression. It's just a tip.